src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
```

Comment:
again Ball and lottery have nothing in common from the conceptual point of view, so they should not extend each other, but be separate classes